### PR TITLE
Feature/sui pta component

### PR DESCRIPTION
**Pull request by zecafa**

### CODE REVIEW

File: components/form/pta/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    width: 100%;
+  }
+
+  &-topbar {
+    width: 100%;
+  }
+
+  &-pta {
+    color: $c-white;
+    padding: 50px;
```
Comment (davidbarna) [inline components/form/pta/src/index.scss]: You should use a $p-* var of the theme

File: components/form/pta/src/index.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    contentWindow.postMessage(formSettings, '*')
+  }
+
+  render () {
+    const { props: { formUrl }, sendSettingsToForm } = this
+
+    return (
+      <div>
+        <h1>PTA Form builder</h1>
```
Comment (davidbarna) [inline components/form/pta/src/index.js]: are you really displaying this title in ALL sites ? No translation ?
I thought it was going to be only an iframe.

Comment (zecafa) [reply components/form/pta/src/index.js]: Upss, you are right!

File: components/form/pta/src/index.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+export const BASE_CLASS = 'sui-FormPta'
```
Comment (jcger) [inline components/form/pta/src/index.js]: why do you need to export this?

File: components/form/pta/src/index.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    const formSettings = {
+      enableDraft,
+      formUrl,
+      exitURLs: {
+        success: redirectOnSuccessUrl,
+        error: redirectOnErrorUrl
+      }
+    }
+
+    contentWindow.postMessage(formSettings, '*')
```
Comment (jcger) [inline components/form/pta/src/index.js]: should the targetOrigin be a prop for security reasons?

File: components/form/pta/src/index.js (hunk 3 above)
Comment (midudev) [inline components/form/pta/src/index.js]: You're not using this const. You may want to use it to build the classnames in the `render`. Besides you're exporting it for some reason?

Comment (zecafa) [reply components/form/pta/src/index.js]: I don't need to export it ;-)

Comment (zecafa) [reply components/form/pta/src/index.js]: I don't need to export it ;-)

File: components/form/pta/src/index.js (hunk 4 above)
Comment (zecafa) [reply components/form/pta/src/index.js]: 👍

File: components/form/pta/src/index.scss (hunk 1 above)
Comment (zecafa) [reply components/form/pta/src/index.scss]: 👍

File: components/form/pta/src/index.js (hunk 2 above)
Comment (carlesnunez) [reply components/form/pta/src/index.js]: As David said. Is it going to be an iframe wrapper? or are you going to add the title too?

**Review by carlosvillu — APPROVED** [review]

**Pull request merged**